src/DIRAC/Interfaces/API/Dirac.py

```diff
@@ -1772,7 +1772,7 @@ def getJobStatus(self, jobID):
     if self.jobRepo:
       self.jobRepo.updateJobs(repoDict)
     for job, vals in siteDict.items():  # can be an iterator
-      result[str(job)].update(vals)
+      result[job].update(vals)
```

Comment:
Can you explain why this "fixed" in the first place (https://github.com/DIRACGrid/DIRAC/commit/dfbb53232dba40aff4bf0f04802877c51d078ab1) and why it doesn't need to be now?

Reply:
First fix resolve error described in previous hackathon https://trello.com/c/y7TgEuF7/40-multi-vo-tests. Now after the last lot of changes, this key returns correctly, so I canceled the last adjustment.